vyper/ast/nodes.py

```diff
@@ -728,6 +728,10 @@ def validate(self):
 class Log(Stmt):
     __slots__ = ("value",)
 
+    def validate(self):
+        if not isinstance(self.value, Call):
```

Comment:
Is there a test that covers this?

Reply:
yea, it's in the structure exception tests somewhere

Reply:
hmm, i thought i saw it before but maybe i hallucinated, so i added one here https://github.com/charles-cooper/vyper/pull/25/commits/9ed427fec0cd869ab253cde5cbe022bb6fa3684a